data/web/js/site/quarantine.js

```diff
@@ -15,10 +15,11 @@ jQuery(function($){
         {"name":"id","type":"ID","filterable": false,"sorted": true,"direction":"DESC","title":"ID","style":{"width":"50px"}},
         {"name":"qid","breakpoints":"all","type":"text","title":lang.qid,"style":{"width":"125px"}},
         {"name":"sender","title":lang.sender},
+        {"name":"subject","title":lang.subj, "type": "text"},
         {"name":"rcpt","title":lang.rcpt, "breakpoints":"xs sm md", "type": "text"},
         {"name":"virus","title":lang.danger, "type": "text"},
         {"name":"score","title": lang.spam_score, "type": "text"},
-        {"name":"subject","title":lang.subj, "type": "text"},
+        {"name":"notified","title":lang.notified, "type": "text"},

```

Comment:
This should be small and hidden.

Reply:
Hi @andryyy 
with `hidden` you mean like qid? This one shown when expanding.

How to make it smaller? The culumn takes it place, which he needs.